docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -128,7 +130,11 @@ Allows setting a sort order for the result. Defaults to `start_time`, i.e. sorti
 (Optional, string)
 Sort order. Valid values are `asc` for ascending and `desc` for descending order. Defaults to `asc`, meaning ascending order.
 
-NOTE: The pagination parameters `size`, `order`, and `sort` are not supported when using `verbose=false` and the sort order for
+`after`::
+(Optional, string)
+Offset identifier to start pagination from as returned by the `next` field in the response body.
```

Comment:
I think it is worth spending a few lines explaining that the underlying list can change and how pagination works in case of deletes and additions during pagination (i.e., you are sure to not miss any that were in the list originally and have not been deleted during pagination and you may see snapshots added during pagination).

Reply:
++ added a few lines on this